### PR TITLE
Add job to load archive data about exchange rate from API.

### PR DESCRIPTION
**Pull request by MrHurniak**
Add endpoint to get list of missed dates in DB.

### CODE REVIEW

File: src/main/java/com/demo/entrepreneur/service/CurrencyService.java [hunk 1]
```diff
@@ -36,17 +46,55 @@ public CurrencyService(ExchangeRateRepository rateRepository, ExchangeRateMapper
     }
 
     public void updateExchangeRates() {
-        LOGGER.info("Call currency api({}) to update currency.", apiUrl);
+        log.info("Call currency api({}) to update currency.", apiUrl);
 
         ResponseEntity<ExchangeRateDto[]> responseEntity = restTemplate.getForEntity(apiUrl, ExchangeRateDto[].class);
+        log.info("Look here {}", responseEntity.getBody());
         ExchangeRateDto[] rates = responseEntity.getBody();
         List<ExchangeRate> exchangeRates = Stream.of(rates)
```
[the remark this answers is not in the document]
Comment (MrHurniak) [reply src/main/java/com/demo/entrepreneur/service/CurrencyService.java]: @Tordek947 , if u are talking about "Look here" that's my fault, as it was used for debug and I just forgot about this.

[the remark this answers is not in the document]
Comment (kkobziev) [reply src/main/java/com/demo/entrepreneur/service/CurrencyService.java]: fixed

File: src/main/java/com/demo/entrepreneur/schedule/impl/LoadArchivedRateJob.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import com.demo.entrepreneur.service.CurrencyService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class LoadArchivedRateJob implements ScheduledJob {
+
```
[the remark this answers is not in the document]
Comment (kkobziev) [reply src/main/java/com/demo/entrepreneur/schedule/impl/LoadArchivedRateJob.java]: fixed

File: src/main/java/com/demo/entrepreneur/schedule/impl/CurrencyRateUpdateJob.java [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
 
+import com.demo.entrepreneur.schedule.ScheduledJob;
 import com.demo.entrepreneur.service.CurrencyService;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
 @Service
-public class CurrencyRateUpdateJob {
+public class CurrencyRateUpdateJob implements ScheduledJob {
```
[the remark this answers is not in the document]
Comment (kkobziev) [reply src/main/java/com/demo/entrepreneur/schedule/impl/CurrencyRateUpdateJob.java]: fixed

**Pull request merged**